src/test/java/tutorspet/logic/parser/EditAttendanceCommandParserTest.java

```diff
@@ -27,43 +32,94 @@
     @Test
     public void parse_missingParts_failure() {
         // no class index specified
-        assertParseFailure(parser, " l/1 s/1 w/1 p/70", MESSAGE_INVALID_FORMAT);
+        assertParseFailure(parser, " "
+                + PREFIX_LESSON_INDEX + "1" + " "
+                + PREFIX_STUDENT_INDEX + "1" + " "
+                + PREFIX_WEEK + "1" + " "
+                + PREFIX_PARTICIPATION_SCORE + "70", MESSAGE_INVALID_FORMAT);
```

Comment:
70 feels like a magic number... Do u mind changing it to the constant `VALID_PARTICPATION_SCORE_80`?

Reply:
same for the ones below... and the 1000 could be change to `INVALID_PARTICIPATION_SCORE_101`...